assets/src/scss/pages/post/_author-block.scss

```diff
@@ -1,86 +1,123 @@
 .author-block {
   overflow: hidden;
-  display: inline-block;
-  margin-top: $space-xs;
-  margin-bottom: $space-md;
+  background: $grey-10;
+  padding: 24px 16px;
+  border-radius: 4px;
+  margin-bottom: 24px;
 
   @include large-and-up {
-    margin-top: $space-md;
-    margin-bottom: $space-lg;
+    max-width: 776px;
   }
 
-  @include medium-and-up {
-    display: inline-flex;
-    align-items: stretch;
+  @include x-large-and-up {
+    max-width: 736px;
   }
-}
 
-.author-block-info {
-  background: $grey-20;
-  padding: $n35 $n20;
+  > .author-block-image {
+    display: none;
+  }
 
-  @include medium-and-up {
-    padding: $n25;
-    position: relative;
-    z-index: 1;
+  .author-block-info-title {
+    font-size: 20px;
+    font-weight: bold;
+    text-align: center;
+    margin-bottom: 16px;
   }
-}
 
-.author-block-info-title {
-  font-size: $font-size-md;
-  font-weight: bold;
-  text-transform: uppercase;
+  .author-block-image {
```

Comment:
Do we have to move this selector inside? It already has `.author-block` in the name, so we're not really gaining anything by nesting it. If it's very nested and you get a link like this https://github.com/greenpeace/planet4-master-theme/blob/a150337b5e34805ff2cc00e195c2d253e3314432/assets/src/scss/pages/post/_author-block.scss#L68, it's not trivial to locate all the lines to construct the selector if it goes a few levels deep.

Reply:
Whereas before all these were just at the top level, and it's only a few lines up to know what the line does. This may not be obvious for you now because you just worked on the code, but if you're searching through a lot of code and don't have much context it can make a big difference.

Reply:
Compare it to the current version. This is the deepest nested line I found and it's still easy to see what the selector is without scrolling. https://github.com/greenpeace/planet4-master-theme/blob/master/assets/src/scss/pages/post/_author-block.scss#L75